helm-chart/binderhub/templates/deployment.yaml

```diff
@@ -1,9 +1,14 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: binder
 spec:
   replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: binder
+      component: binder
+      release: {{ .Release.Name }}
```

Comment:
note btw that I left out some labels here, it is not considered good practice selecting on some, such as `heritage: tiller` which is good because it is gone in helm 3 :D